crates/rpc-types-eth/src/filter.rs

```diff
@@ -38,6 +38,14 @@ impl BloomFilter {
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 pub struct FilterSet<T: Eq + Hash>(HashSet<T>);
 
+impl FilterSet<Topic> {
```

Comment:
this isn't quite right:

https://github.com/alloy-rs/alloy/blob/72a12eab59df688abce53da845ee66dcc1060c86/crates/rpc-types-eth/src/filter.rs#L147-L147

we basically want `impl Topic` here

`FilterSet<Topic>` is `FilterSet<FilterSet<B256>>`